app/views/v2/lookup.scala.html

```diff
@@ -45,6 +45,10 @@ <h1 class="govuk-heading-xl" id="pageHeading">
         @{if(isUKMode) messages("lookupPage.heading.ukMode") else messages("lookupPage.heading")}
       </h1>
 
+      @if(messages.isDefinedAt("lookupPage.afterHeadingText")) {
```

Comment:
Interesting.  What not pass an `afterHeadingMessage: Option[String]` parameter into the view (e.g. like `message`)?   Or is this the ALF way of doing "optional" content?

Reply:
The message field is never used it was discussed with the team when we asked for permission to do this.
It appears to be a relic, something that was tried, never used or implemented fully.
There is no way to activate it from config.

But, removing here would be beyond the scope of what we agreed. They may well remove it.
HTH

Reply:
I've taken it from the spike verbatium.  There is some magic that happens with messages where they get merged up from config.  This seems to be the less intrusive mechanism of showing or hiding this field if a key/value pair does not exist in the message bundle (after the RemoteMessageAPI does it's thing).  I will take a steer from ALF here though.

Reply:
My comment was more about the way we're conditionally rendering this new field - by checking for a definition in the messages file.  It's unusual, that's all.